### ISSUE TITLE
License is not open source

### ISSUE THREAD

**Issue by kappapiana** [g0v-it/voting-tools#1]
I note that while the license is clearly drafted and very permissive, it does not qualify as open source, as -- among other things -- it bears clear limitations as to by whom and for which purposes the software can be used and this is against both Freedom #0, and rules #5 and #6 of the OSD. Therefore, I submit that changing the licensing would be necessary to call this software open source and to facilitate its adoption.

I suggest to pick one of the already approved licenses in the OSI roster, in order to simplify the parsing of the license and to reduce proliferation. https://opensource.org/licenses

Depending on the needs and attitude of the project, I am available to advise on a complimentary basis which might suit the requirements better.

**Comment by smaffulli** [comment]
I agree with @kappapiana: choosing a non-standard, non-open-source license damages the possibility of the project to gain adoption. As a coincidence, today I stumbled upon this article from 2020, exactly about the effect on community building for projects with non-open-source licenses (despite the good intentions) https://www.techrepublic.com/article/how-jshint-learned-the-hard-way-not-to-use-ethical-source-licensing/

**Comment by ecow** [comment]
In my opinion, the author is free to use the license he wants for his work. The published software is clearly "open source" according to Oxford Dictionary because it is made available to anyone. Clearly is not an open source license according to OSD nor a FOSS according to Free Software Foundation definition. I agree that such type of license damages the possibility of the project gaining large adoption, but this does not matter to the author. OSD "rules"  apply only for OSD licenses.

**Comment by smaffulli** [comment]
@ecow you're spreading FUD by quoting the Oxford dictionary in this context. Open Source in the context of software has not only a well defined meaning in the Open Source Definition but also a well recognized authority by a wide array of institutions. From the Italian government to many US federal agencies to UK government, industry standards, the definition of "Open Source" is recognized as the one provided by the Open Source Initiative. You'll find samples with links of what I'm saying on https://opensource.org/authority. 

Developers are free to choose the license they want, nobody here is arguing their freedom of choice. Just avoid muddying the waters and confuse users by calling Open Source something that the industry has agreed is not.
